docs/content/authors.md

```diff
@@ -587,7 +587,7 @@ put them back in again.` >}}
   * Leroy van Logchem <lr.vanlogchem@gmail.com>
   * Zsolt Ero <zsolt.ero@gmail.com>
   * Lesmiscore <nao20010128@gmail.com>
-  * ehsantdy <ehsan.tadayon@arvancloud.com>
+  * ehsantdy <ehsan.tadayon@arvancloud.ir>
```

Comment:
You'll need to add the new email as well as the old email here so

```
ehsantdy <ehsan.tadayon@arvancloud.com> <ehsan.tadayon@arvancloud.ir> 
```

Otherwise the auto script will add it straight back again.

If you don't want your old email to appear you will need to add it to

`bin/.ignore-emails`

Reply:
Hi Nick, thanks for your reply. I fix them and make backend docs. sorry for my mistake. I will push again.